lib/system/memalloc.nim

```diff
@@ -188,8 +188,8 @@ when hasAlloc and not defined(js):
     cast[ptr T](realloc(p, T.sizeof * newSize))
 
   proc dealloc*(p: pointer) {.noconv, compilerproc, rtl, benign, raises: [], tags: [].} =
-    ## Frees the memory allocated with `alloc`, `alloc0` or
-    ## `realloc`.
+    ## Frees the memory allocated with `alloc`, `alloc0`,
+    ## `realloc` or `create`.
```

Comment:
Maybe link those?

Reply:
ok to defer IMO; let's just implement https://github.com/nim-lang/RFCs/issues/125 instead; then you'll just write
```
realloc_ or create_.
```

Reply:
make sense